dali/operators/image/crop/bbox_crop.cc

```diff
@@ -43,7 +43,7 @@ void CollectShape(std::vector<TensorShape<>> &v,
                   const OpSpec& spec,
                   const workspace_t<CPUBackend>& ws,
                   int ndim) {
-  int batch_size = spec.GetArgument<int>("batch_size");
+  int batch_size = spec.GetArgument<int>("max_batch_size");
```

Comment:
Shouldn't it introduce batch_size argument for the CollectShape function instead of using max_batch_size?